apps/site/components/withNavBar.tsx

```diff
@@ -10,19 +11,30 @@ import { useSiteNavigation } from '@/hooks';
 import { useRouter, usePathname } from '@/navigation.mjs';
 import { availableLocales } from '@/next.locales.mjs';
 
+import Button from './Common/Button';
+
 const WithNavBar: FC = () => {
   const { navigationItems } = useSiteNavigation();
   const { resolvedTheme, setTheme } = useTheme();
   const { replace } = useRouter();
   const pathname = usePathname();
 
+  const skipToContent = classNames(
+    'absolute left-0 top-0 m-3 -translate-y-16 bg-blue-500 p-3 transition-transform focus:translate-y-0 focus:outline-none'
+  );
+
```

Comment:
```suggestion
```

Since there is no conditional class added, there is no need to use `classNames`, you can write styles directly